students/Kenneth_Murray/lesson3/string_formatting/str_format_Lab.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+"""String Formatting Lesson 3 task1"""
+file_data = ( 2, 123.4567, 10000, 12345.67)
+file_string = 'file_{} :   {}, {}, {}'
+file_n = '{0:0{width}}'.format(file_data[0], width=3)
+round2 = round(file_data[1],2)
+sci_d2 = '{:.2E}'.format(file_data[2])
+flt_sci3 = '{:.2E}'.format(float(file_data[3]))
+new_file_string = file_string.format(file_n,round2,sci_d2,flt_sci3)
```

Comment:
best to add your `.format()` piece to the actual string you are inputing variables in.  Otherwise another programmer will have no idea how that stuff fits in the string.